misk/web/modules/healthcheck/src/mainMenu/components/MenuComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+const { Link } = require('frint-router-react');
+import { MenuItem, MenuDivider } from '@blueprintjs/core';
+
+export namespace MenuComponent {
+  export interface Props {
+    match: {
+      isExact: boolean,
+      params: {
+        moduleID: string
+      },
+      url: string
+    }
+  }
+}
+
+class MenuComponent extends React.Component<MenuComponent.Props> {
+  render() {
+    if (false) {
+      return (
+        <Link
+            to="/"
+            className="nav-item is-tab"
+            activeClassName="is-active"
+            exact
+          >
+          <MenuItem icon="new-object" text="New object"/>
+        </Link>
+      )
+    }
+
+    return (
+      <div>
+        <Link to="/log4j">Another tests</Link>
```

Comment:
nit: log4J

Reply:
Haven't dug deep into building Healthcheck yet, just copied it over from Config since Webpack was complaining about having an incomplete module to build.